lib/trip_dispatcher.rb

```diff
@@ -19,20 +20,51 @@ def find_passenger(id)
       return @passengers.find { |passenger| passenger.id == id }
     end
 
+    def find_driver(id)
+      Driver.validate_id(id)
+      return @drivers.find { |driver| driver.id == id }
+    end
+
     def inspect
-      # Make puts output more useful
+      #Make puts output more useful
       return "#<#{self.class.name}:0x#{object_id.to_s(16)} \
               #{trips.count} trips, \
               #{drivers.count} drivers, \
               #{passengers.count} passengers>"
     end
 
+    def request_trip(passenger_id)
+      assigned_driver = @drivers.find(ifnone = nil) do |driver|
```

Comment:
Being explicit is fine but `ifnone` defaults to `nil`, so if you don't need to include it if you don't want to.
```suggestion
      assigned_driver = @drivers.find do |driver|
```